test/Test/Iris/Interactive/Question.hs

```diff
@@ -0,0 +1,36 @@
+module Test.Iris.Interactive.Question (questionSpec) where
+
+import Test.Hspec (Spec, describe, it, shouldReturn)
+
+import Control.Monad (forM_)
+import Data.Text (unpack)
+
+import Iris.Interactive.Question (
+    -- under test
+    YesNo (..),
+    parseYesNo,
+ )
+
+questionSpec :: Spec
+questionSpec =
+    describe "Question - parse YesNo" $ do
+        describe "should parse to Just Yes" $ do
+            forM_ ["y", "Y", "YES", "yes", "Yes"] $
+                \m -> do
+                    it (unpack m) $ do
+                        pure (parseYesNo m) `shouldReturn` Just Yes
+
+        describe "should parse to Just No" $ do
+            forM_ ["n", "N", "NO", "no", "No"] $
```

Comment:
Let's add one more test case just to be extra sure that all inputs are actually handled 🙂 

```suggestion
            forM_ ["n", "N", "NO", "no", "No", "nO"] $
```